src/xls.rs

```diff
@@ -455,10 +461,60 @@ impl<RS: Read + Seek> Xls<RS> {
     }
 }
 
+/// https://learn.microsoft.com/en-us/openspecs/office_file_formats/ms-xls/4d6a3d1e-d7c5-405f-bbae-d01e9cb79366
+struct Bof {
+    /// Binary Interchange File Format
+    biff: u8,
+}
+
+/// BOF [MS-XLS] 2.4.21
+fn parse_bof(r: &mut Record<'_>) -> Result<Bof, XlsError> {
+    let mut dt = 0;
+    let biff_version = read_u16(&r.data[..2]);
+
+    if r.data.len() >= 2 {
+        dt = read_u16(&r.data[2..]);
+    };
+
+    let mut biff = match biff_version {
+        // BIFF8
+        0x0600 => 8,
+        // BIFF5
+        0x0500 => 5,
+        // BIFF4
+        0x0400 => 4,
+        // BIFF3
+        0x0300 => 3,
+        // BIFF2
+        0x0200 | 0x0002 | 0x0007 => 2,
+        _ => {
+            if r.data[4..].len() > 6 {
```

Comment:
This should probably be checked for all Bofs? Spec says that is `r.data.len() == 16`?

Reply:
Agree, although I'm not sure if we need this check at all. Initially I've copied it from sheetjs, but now I don't see a purpose to be honest. Especially if I cannot write a test (I don't have an example xls file) to hit this line. What do you think?